src/pages/K8s.vue

```diff
@@ -198,6 +201,16 @@ export default {
       this.$data.integrations = integrations;
     });
     ipcRenderer.send('k8s-integrations');
+    ipcRenderer.on('k8s-integration-extra-info', (event, name, warnings) => {
+      if (warnings.length === 0) {
+        this.$delete(this.integrationWarnings, name);
+      } else {
+        this.$set(this.integrationWarnings, name, warnings);
+      }
+    });
+    this.$nextTick(() => {
```

Comment:
Hmm, why does this need to be on next tick? `k8s-integrations` works okay without.